scripts/vim-patch.sh

```diff
@@ -36,17 +36,17 @@ check_executable() {
 }
 
 get_vim_sources() {
-  check_executable hg
+  check_executable git
 
   echo "Retrieving Vim sources."
   if [[ ! -d ${VIM_SOURCE_DIR} ]]; then
     echo "Cloning Vim sources into '${VIM_SOURCE_DIR}'."
-    hg clone https://code.google.com/p/vim "${VIM_SOURCE_DIR}"
+    git clone https://github.com/vim/vim.git "${VIM_SOURCE_DIR}"
     cd "${VIM_SOURCE_DIR}"
   else
     echo "Updating Vim sources in '${VIM_SOURCE_DIR}'."
     cd "${VIM_SOURCE_DIR}"
-    hg pull --update &&
+    git pull &&
```

Comment:
Should probably be a `git fetch --all`. It doesn't really matter what is actually checked out in the workspace, the script can pull from `origin/master`.